app/javascript/file-upload.js

```diff
@@ -0,0 +1,66 @@
+import MOJFrontend from '@ministryofjustice/frontend'
+import $ from 'jquery'
+window.$ = $
+
+export default function initFileUpload() {
+    MOJFrontend.MultiFileUpload.prototype.getFileRowHtml = function (file) {
+        var html =
+            `<tr class="govuk-table__row moj-multi-file-upload__row">
+            <td class="govuk-table__cell moj-multi-file-upload__filename">
+                <span class="moj-multi-file-upload__filename"> ${file.name}</span>
+                <span class="moj-multi-file-upload__progress">(0%)</span></td>
+            <td class="govuk-table__cell moj-multi-file-upload__actions">
+                <a class="remove-link moj-multi-file-upload__delete" href="#0" value="${file.name}">Remove 
+                <span class="govuk-visually-hidden">${file.name}</span>
+                </a>
+            </td>
+        </tr>`
+
+        return html;
+    };
+
+    MOJFrontend.MultiFileUpload.prototype.onFileDeleteClick = function (e) {
+        e.preventDefault(); // if user refreshes page and then deletes
+        var button = $(e.currentTarget);
+        $.ajax({
+            url: `${this.params.deleteUrl}?resource_id=${button[0].attributes[2].value}`,
```

Comment:
```
button[0].attributes[2].value
```
what does the 2 mean? just seems dangerous

Reply:
Possibly a better way of doing this but need to work with @imtiazAhmd to get this working as we would like

Reply:
Resolved in a cleaner fashion